lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/StateMachineInitializer.java

```diff
@@ -26,25 +29,35 @@ public class StateMachineInitializer {
             new ObjectMapper(new YAMLFactory()).configure(STRICT_DUPLICATE_DETECTION, true);
     private static final ObjectMapper om = new ObjectMapper();
     private Map<String, State> journeyStates;
-    private Map<String, NestedJourneyDefinition> nestedJourneyDefinitions;
+    private final HashMap<String, NestedJourneyDefinition> nestedJourneyDefinitions =
```

Comment:
It's usually better to have a specific collection implementation, unless you particularly need certain characteristics.

Could this just be `Map<String, NestedJourneyDefinition> nestedJourneyDefinitions = new HashMap<>();`?